server/internal/attr/conventions.go

```diff
@@ -705,5 +709,8 @@ func SlogClickhouseQueryDurationMs(v float64) slog.Attr {
 	return slog.Float64(string(ClickhouseQueryDurationMsKey), v)
 }
 
-func SlogURL(v string) slog.Attr        { return slog.String("url", v) }
-func SlogRegistryID(v string) slog.Attr { return slog.String("registry_id", v) }
+func SlogURL(v string) slog.Attr             { return slog.String("url", v) }
+func SlogRegistryID(v string) slog.Attr      { return slog.String("registry_id", v) }
+func SlogExternalMCPID(v string) slog.Attr   { return slog.String("external_mcp_id", v) }
+func SlogExternalMCPSlug(v string) slog.Attr { return slog.String("external_mcp_slug", v) }
+func SlogExternalMCPName(v string) slog.Attr { return slog.String("external_mcp_name", v) }
```

Comment:
Follow the conventions on how to add stuff to this file please - declare keys in the const black and create OTEL + Slog variants. Mentioned it in another PR.